src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -25,13 +27,13 @@ interface PropsType {
 }
 
 const sparkleAvatars = [
-  "default-profile-pic-1.png",
-  "default-profile-pic-2.png",
-  "default-profile-pic-3.png",
-  "default-profile-pic-4.png",
+  "/avatars/default-profile-pic-1.png",
+  "/avatars/default-profile-pic-2.png",
+  "/avatars/default-profile-pic-3.png",
+  "/avatars/default-profile-pic-4.png",
 ];
```

Comment:
This seems like it should probably be extracted into `src/settings` and renamed something like `DEFAULT_AVATARS`/`DEFAULT_AVATAR_URLS` or similar.

Reply:
Changed in: https://github.com/sparkletown/sparkle/commit/6cdbdb2d81f61c67a0665f91bf358d09fc89bfbe